codegen/luci-codegen/src/Driver.cpp

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "Codegen.h"
+#include "luci/Importer.h"
+#include "luci/CircleExporter.h"
+#include "luci/CircleFileExpContract.h"
+
+#include <boost/filesystem.hpp>
```

Comment:
@seanshpark 
I noticed that compiler components try to minimize dependencies from `Boost` (and other third party libraries).
If I understand it correctly this is for portability reasons: we do not need to provide libraries with compiled toolchain.

As you can see this code uses Boost filesystem module, I can remove dynamic dependency by building compiler using static linking. Is it ok solution?

Otherwise, do we have some filesystem related submodule in compiler that I can use? (like `arser` instead of `boost::program_options`)

Reply:
>  I can remove dynamic dependency by building compiler using static linking. Is it ok solution?

This is nice not to release with external libs :) But this may grow bin size but I don't think not that much in this case...

> do we have some filesystem related submodule in compiler that I can use?

You used 
```cxx
  boost::filesystem::path output_package_dir(output_package_path);
  if (!boost::filesystem::exists(output_package_dir))
  {
    boost::filesystem::create_directory(output_package_dir);
  }
  boost::filesystem::path output_model_path = output_package_dir / "model.circle";
```
which can be implemented in standard APIs... but `output_package_dir / "model.circle";` need an `if` line...

So, my thoughts are something like this;
1) If you can and willing to maintain `codegen` root folder, I think it is up to you
2) `maintain` will require responsibility of release tasks also. This may have narrow time slot.
3) as we are not developing cross-platform product, that is, we (currently) release only for Ubuntu, we can provide a small functions that acts like boost APIs, in this case (as I know, boost also covers platform dependent things)
4) when these libs are used several places, we can then make separate module in `codegen/` or in `compiler/`
5) when it's time we can use c++17, then these can be refactored using std libs :)